main.go

```diff
@@ -30,6 +30,7 @@ func main() {
 		RegisterBackupItemAction("velero.io/csi-snapshotter", newCSISnapshotter).
 		RegisterBackupItemAction("velero.io/volumesnapshot-backupper", newVolumeSnapshotBackupItemAction).
 		RegisterBackupItemAction("velero.io/volumesnapshotclass-backupper", newVolumesnapshotClassBackupItemAction).
+		RegisterBackupItemAction("velero.io/volumesnapshotcontent-backupper", newVolumeSnapContentBackupItemAction).
 		RegisterRestoreItemAction("velero.io/csi-restorer", newCSIRestorer).
 		RegisterRestoreItemAction("velero.io/volumesnapshotclass-restorer", newVolSnapClassRestorer).
 		RegisterRestoreItemAction("velero.io/volumesnapshotcontents-restorer", newVSCRestorer).
```

Comment:
one tiny nit: I'd make the plugin name here `velero.io/volumesnapshotcontent-restorer` (remove the `s` after `volumesnapshotcontent`) to be consistent.

Reply:
fixed!